lib/user.js

```diff
@@ -60,9 +60,16 @@ module.exports = function (restClient) {
   }
   module.me = function (customerToken) {
     const url = `user/me?token=${customerToken}`
-    return restClient.get(url).then((data)=> {
+    return restClient.get(url).then((data) => {
       return getResponse(data);
     });
   }
+  module.setNewPassword = function (passwordData) {
+    url += `setNewPassword`;
```

Comment:
this should be /user/password

Reply:
it is, url = 'user/'

Reply:
and user/setNewPassword is valid url according to docs in jira task